### DISCUSSION TITLE
post/ifkakao2024-document-ai/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#112] [Announcements]
# post/ifkakao2024-document-ai/

카카오페이손해보험에서는 보험금 청구를 위한 일 평균 3,000장의 문서를 사람이 아닌 Document AI가 처리하고 있습니다. 어떤 모델로 어떻게 구현했는지에 대한 방법을 공유합니다.

https://tech.kakaopay.com/post/ifkakao2024-document-ai/

**Comment by jws1837** [comment]
너무 이해 잘되고 재밌게 글 읽었습니다. 감사합니다.

**Comment by gimhaemulgae** [comment]
알찬 내용 잘 보았습니다.
혹시 1장당 인식 속도는 얼마나 걸릴까요?
아무래도 고객이 늘어나고 동시 추론이 늘어나게 되면 속도가 느려질 수 있는데, 이에 대한 고민이 있으셨을 것 같습니다.
